Dom.zavd 1/Salon.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+class Car
+{
+    private string Brand { get; set; }
+    public Car(string _Brand)

```

Comment:
Wrong naming, it should not use underscore here

Reply:
See https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions